src/guide/typescript/composition-api.md

```diff
@@ -22,6 +22,22 @@ props.bar // number | undefined
 
 This is called "runtime declaration", because the argument passed to `defineProps()` will be used as the runtime `props` option.
 
+Prop types can also be explicitly declared when using runtime declaration:
+
+```vue
+<script setup lang="ts">
+const props = defineProps({
+  foo: {
+    type: String as PropType<string>,
```

Comment:
This is a redundant typing as the type `string` can already be inferred from the `String` constructor (same for `number` and `Number` below). I recommend showing an example where `PropType` is necessary. For example:

```typescript
type: Array as PropType<string[]>
```

Reply:
Maybe reuse the `PropType` example from the Options API page, ported to `<script setup>`?

Reply:
@tony19 I know, but for `Object` that wouldn't be the case. I just wanted to show how it works, as a 1:1 copy of the previous example.